enterprise/ha/src/test/java/org/neo4j/kernel/impl/ha/ParallelLifecycle.java

```diff
@@ -80,7 +82,9 @@ public void shutdown() throws Throwable
         perform( Lifecycle::shutdown );
     }
 
-    private void perform( Action action ) throws InterruptedException
+    private static final AtomicBoolean FAILING = new AtomicBoolean();
```

Comment:
why do we need this field? and it looks it like to be in the center on attention and find a good place in the middle of the class file :)